sys-kernel/linux-firmware/linux-firmware-99999999.ebuild

```diff
@@ -40,6 +40,7 @@ RESTRICT="binchecks strip test
 BDEPEND="initramfs? ( app-alternatives/cpio )
 	compress-xz? ( app-arch/xz-utils )
 	compress-zstd? ( app-arch/zstd )
+	python
```

Comment:
Indeed, this isn't right, the ebuild will have to start using [python-any-r1](https://projects.gentoo.org/python/guide/any.html). I don't think you should feel obligated to do that though, rather just someone (if you're not willing/able) should do it before merging obviously.